hathor/transaction/resources/push_tx.py

```diff
@@ -150,7 +151,17 @@ def render_POST(self, request: Request) -> bytes:
         if not body_content:
             return error_ret
 
-        data = json_loadb(body_content)
+        try:
+            data = json_loadb(body_content)
+        except JSONDecodeError as exc:
+            return json_dumpb({
+                'success': False,
+                'message': str(exc),
+                'can_force': False,
+            })
+
+        if not isinstance(data, dict):
+            return error_ret
```

Comment:
Should we add this error to the API docs?

Reply:
We usually don't add basic verification errors in the API docs. I mean, users are not following the documentation and it can generate a multitude of errors. What do you think about this approach?

Reply:
It seems ok for me, but I didn't really get what you mean by "users are not following the documentation and it can generate a multitude of errors.".

Another thing I noticed after taking a second look on the Docs is that it states that the Body for `push_tx` should be a "string", but the code is expecting an object with a `hex_tx` key, right?

Reply:
You're right. I think the documentation is wrong. It should say something like "a JSON with the following format: `"{ "hex_tx": "[HEX]"}"`.

Reply:
Fixed! Can you review again, please? @luislhl

Reply:
Thanks!